cachito/workers/config.py

```diff
@@ -3,16 +3,41 @@
 import tempfile
 import logging
 
+import kombu
```

Comment:
@lcarva this dependency comes from Celery, but since I'm directly importing it, should it be part of the requirements.txt file?

Reply:
Yes. I think so since it's now a direct dependency.